AcademyApp/Views/LecturesCollectionViewCell.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  LecturesCollectionViewCell.swift
+//  AcademyApp
+//
+//  Created by Peter Sevcik on 21/06/2019.
+//  Copyright © 2019 Peter Sevcik. All rights reserved.
+//
+
+import UIKit
+
+class LecturesCollectionViewCell: UICollectionViewCell {
+
+    var lectureName: String? {
+        didSet {
+            setup()
+        }
+    }
+
+    @IBOutlet private weak var lectureImageView: UIImageView!
+    @IBOutlet private weak var lectureNameLabel: UILabel!
+    @IBOutlet private weak var checkmarkImageView: UIImageView!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        setup()
+    }
+}
+
+// MARK: - Public setup methods
+
+extension LecturesCollectionViewCell {
+    func configure() {
```

Comment:
there is mix of public property config method setup etc
I would guess bc its lecture cell (no matter if collection view or table view) to configure(with lecture: Lecture) - data  model representing lecture and in that method set title
for reusing cell or if not set lecture keep image nil and title empty

Reply:
LecturesCollectionViewCell was replaced by LecturesTableViewCell.